src/main/java/org/protege/editor/owl/server/http/ServerProperties.java

```diff
@@ -21,6 +21,7 @@
      */
     public static final String EVS_HISTORY_FILE = "evshistory_file";
     public static final String CON_HISTORY_FILE = "conhistory_file";
+    public static final String ARCHIVE_ROOT = "root-archive";
```

Comment:
total nit: I'd make this name `root_archive` to make life easier on the sysadmin